internal/generate/clusterserviceversion/bases/definitions/ast.go

```diff
@@ -31,6 +31,7 @@ func (g generator) getMarkedChildrenOfField(root markers.FieldInfo) (map[string]
 	for len(nextFields) > 0 {
 		fields := []*fieldInfo{}
 		for _, field := range nextFields {
+			field := field
```

Comment:
-1 to reassigning to a local variable in the loop

Reply:
@jmrodri this is not reassignment of a local var...   This is resolving "Using reference to loop iterator variable" as identified as mistake 1 in the [CommonMistakes wiki](https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable).

Reply:
technically it is the creation of a new variable with the same name assigned to the value of the original "field", but allows for the dereferencing of an address to the correct value which the original "field" does not.

Reply:
without this fix... 
```
internal/generate/clusterserviceversion/bases/definitions/ast.go:71:29: Using the variable on range scope `field` in function literal (scopelint)
							pathSegments: append(field.pathSegments, segment),
```